lua-tarantool-scm-1.rockspec

```diff
@@ -14,10 +14,10 @@ description = {
 }
 
 dependencies = {
-  "lua ~> 5.1",
+  "lua >= 5.1",
```

Comment:
This change does not look related to others. Can you clarify it and, if it is really necessary, move to its own commit?

Reply:
So far I have returned it back, but I tested several versions, and it worked.

Reply:
Okay. Please, squash commits and force-push.